shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/framework/container/compose/DockerContainerComposer.java

```diff
@@ -65,7 +65,7 @@ public DockerContainerComposer(final DatabaseType databaseType, final String sto
             storageContainerConfig = StorageContainerConfigurationFactory.newInstance(databaseType);
         }
         storageContainer = getContainers().registerContainer((DockerStorageContainer) StorageContainerFactory.newInstance(databaseType, storageContainerImage,
-                "", storageContainerConfig));
+                "", StorageContainerConfigurationFactory.newInstance(databaseType)));
```

Comment:
create mysql 8 container will volumn another `my.cnf`, relation code is at 61~64 line at this file

Reply:
fixed